functions/pubsub/index.js

```diff
@@ -1,104 +0,0 @@
-/**
- * Copyright 2016, Google, Inc.
```

Comment:
@labtopia @stew-r if memory serves, we've gotten past feedback about people wanting to know how to publish Pub/Sub messages via GCF.

Personally, I'm +1 to deleting this sample though - at least until @grant and I figure out a plan for GCF samples that are _not_ primarily for the GCF docs.

Reply:
We do have an outstanding bug about this. If the sample answers their question, I would be in favor of putting it in the Pub/Sub calling doc. WDYT?

b/132859769

Reply:
On second thought - would a sample like [this one](https://github.com/googleapis/nodejs-pubsub/blob/master/samples/topics.js#L82) be too generic? (It's not GCF-specific.)

Reply:
I think users want to see a literal example of how to send a message from a Cloud Function. Otherwise, they would just be looking at the Pub/Sub docs.

The first one would have to be updated, though, it refers to `gcloud alpha functions`.

Reply:
Add all pubsub samples back in and opened b/134743180